From 1821c16ca30d793542a0a8d2d3497e4237f70d25 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@jumpnowtek.com>
Date: Wed, 14 Sep 2016 13:53:10 -0400
Subject: [PATCH 1/4] ARM: omap4: wakeupgen: Remove gic parent from usb host

This looks like an oversight from commit 7136d457f36.

The problem shows up on a Gumstix Duovero as inabililty to
record audio correctly from a USB device.

These warnings show up while recording

[   53.319152] retire_capture_urb: 3541 callbacks suppressed
[   58.391052] retire_capture_urb: 3580 callbacks suppressed
[   63.471191] retire_capture_urb: 3604 callbacks suppressed
[   68.572967] retire_capture_urb: 3569 callbacks suppressed
[   73.611175] retire_capture_urb: 3537 callbacks suppressed

Signed-off-by: Scott Ellis <scott@jumpnowtek.com>
---
 arch/arm/boot/dts/omap4.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/boot/dts/omap4.dtsi b/arch/arm/boot/dts/omap4.dtsi
index 5a206c1..8a5628c 100644
--- a/arch/arm/boot/dts/omap4.dtsi
+++ b/arch/arm/boot/dts/omap4.dtsi
@@ -844,14 +844,12 @@
 			usbhsohci: ohci@4a064800 {
 				compatible = "ti,ohci-omap3";
 				reg = <0x4a064800 0x400>;
-				interrupt-parent = <&gic>;
 				interrupts = <GIC_SPI 76 IRQ_TYPE_LEVEL_HIGH>;
 			};
 
 			usbhsehci: ehci@4a064c00 {
 				compatible = "ti,ehci-omap";
 				reg = <0x4a064c00 0x400>;
-				interrupt-parent = <&gic>;
 				interrupts = <GIC_SPI 77 IRQ_TYPE_LEVEL_HIGH>;
 			};
 		};
-- 
2.7.4

